src/main/java/com/lorepo/icplayer/client/module/text/TextView.java

```diff
@@ -8,6 +8,7 @@
 import com.google.gwt.user.client.Element;
 import com.google.gwt.user.client.Window;
 import com.google.gwt.user.client.ui.HTML;
+import com.lorepo.icf.utils.JavaScriptUtils;
```

Comment:
nie będę tylko dla tego dawał declined, ale gdybyś jeszcze musiał coś tu robić to wywal proszę nadmiarowy import.